src/providers/app-config/app-config.ts

```diff
@@ -14,6 +14,10 @@ export class AppConfigProvider {
     backgroundColor: '#ffffff'
   };
 
+  googleAnalyticsId: string = 'UA-129489007-1'; // Key for beta
+  // googleAnalyticsId: string = 'UA-129814222-1';   // Brians test key
```

Comment:
Test key

Reply:
We are actually deploying the test key until we deploy for familiarisation and leaving the proper key commented out. Just so we don't pollute the reports while we are developing